department-of-veterans-affairs/va.gov-cms
# VACMS-7408 claro dropbuttons

## Description

Relates to #7408. Fixes dropbutton behavior in vagovclaro by extending claro libraries. Drupal issue linked in the ticket explains that Drupal handles admin sub-themes differently, apparently, so libraries have to be extended manually.

Some minor styles included to help the hover behavior for dropbuttons remain smooth.

## Testing done
Tested dropbuttons on `/admin/structure/views` and `/admin/content-models/categories` and then clicking into any of the taxonomies, the drop buttons there as well. Verified things worked as expected when draggable or when row weights shown instead. 

Also briefly spot-checked node forms with no dropbuttons to make sure that extending libraries didn't introduce any unexpected regressions... nothing immediately jumped out at me.

## Screenshots
n/a

## QA steps

As user with admin role,
- visit any of the links above (e.g. /admin/content-models/categories) and verify that dropbuttons function as expected (can be clicked for an action or clicked to open a menu of additional actions).

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?

VAgovClaro only, no announcement needed
